examples/fullscreen.rs

```diff
@@ -34,8 +34,8 @@ fn main() {
     event_loop.run(move |event, _, control_flow| {
         control_flow.set_wait();
 
-        match event {
-            Event::WindowEvent { event, .. } => match event {
+        if let Event::WindowEvent { event, .. } = event {
+            match event {
```

Comment:
See [my comment on the last Clippy PR](https://github.com/rust-windowing/winit/pull/2108#pullrequestreview-839703098), I still think there is value in making the examples use `match` instead of `if let`.